hw/top_earlgrey/util/vivado_hook_synth_design_pre.tcl

```diff
@@ -0,0 +1,13 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Change the severity of some messages.
+
+# Abort if the boot ROM init file cannot be found. This is normally just a critical warning
+# which is easily overlooked. The bitstream can still be generated but is not functional.
+set_msg_config -id {[Synth 8-4445]} -new_severity ERROR
```

Comment:
I'm pretty sure the Designcheck 2-3 error we have already covers this, or did you find a way to get a bitstream without seeing the existing error?

Reply:
The check we already have only covers this if we try to enforce the placement of the boot ROM needed for the splicing script, i.e., on the NexysVideo board.

On other boards like the ChipWhisperer one, we don't enforce the placement (the specified BRAM sites are not even available on smaller FPGAs). Hence we don't have any checks there. Users just end up with a bitstream that simply does not work but it's not obvious why. @alphan made this unpleasant experience and I verified it in the process of doing this PR.

Also note this check will abort the build already after synthesis while the placement check is performed later (opt design), meaning this helps to save some time also on failing NexysVideo builds.

Reply:
oh I see. Good to know that we can safely remove the other check one day :-)